KalebIrwin-Power-signoff.md

```diff
@@ -0,0 +1,38 @@
+## Function of the subsystem 
+The power subsystem will provide the necessary energy for the components to operate. 
```

Comment:
Be sure to use "power" and "energy" correctly. In this sentence "power" is probably what is meant.